File: GameAPI/src/main/java/me/clutchy/games/TestListener.java
package me.clutchy.games;

import org.bukkit.Bukkit;
import org.bukkit.Location;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class TestListener implements Listener {

    @EventHandler
    public void onPlayerJoin(PlayerJoinEvent event) {
        event.getPlayer().teleport(new Location(Bukkit.getWorld("Lobby"), 8 , -59 , 0));
    }
}

File: GameAPI/src/main/kotlin/me/clutchy/games/TestListener.kt
package me.clutchy.games

import org.bukkit.Bukkit
import org.bukkit.Location
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

class TestListener: Listener {

    @EventHandler
    fun onPlayerJoin(event: PlayerJoinEvent) {
        event.player.teleport(Location(Bukkit.getWorld("Lobby"), 8.0, -59.0, 0.0))
    }
}
